pandas-stubs/core/frame.pyi

```diff
@@ -1316,9 +1316,14 @@ class DataFrame(NDFrame, OpsMixin):
     ) -> DataFrame: ...
 
     # Add spacing between apply() overloads and remaining annotations
+    # Deprecated in pandas 2.1.0 and later
     def applymap(
         self, func: Callable, na_action: Literal["ignore"] | None = ..., **kwargs
     ) -> DataFrame: ...
```

Comment:
We're removing deprecated methods, so can you delete `applymap()` ?

Reply:
Removed applymap